### PR TITLE
Updating Ruby on Rails version number push

### PR DESCRIPTION
**Pull request by Xiaoming94**
Update to rails 5, solves #244

### CODE REVIEW

**Comment by Vilddjur** [comment]
Seems like you copied the stylesheets folder into itself, so we now have `app/assets/stylesheets/stylesheets/` containing duplicate information?

**Comment by Xiaoming94** [comment]
I pushed it accidentally, can remove it now

File: config/application.rb [hunk 1]
```diff
@@ -25,6 +25,6 @@ class Application < Rails::Application
     config.i18n.default_locale = :sv
     config.i18n.available_locales = [:en, :sv]
     # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
+    #config.active_record.raise_in_transactional_callbacks = true
```
Comment (malmers) [inline config/application.rb]: This line should be removed

**Comment by Xiaoming94** [comment]
The updates are done, and the code are ready to be merged.

Many thanks to @malmers , @lindskogen and @Vilddjur Får all the help and humble guidance!

**Pull request merged**